web/src/Components/Optimization/OptimizationRunner.tsx

```diff
@@ -12,7 +12,14 @@ import { AuthContext, ParticleSizeContext } from '../../Context'
 import EditProducts from '../Common/EditProducts'
 import useLocalStorage from '../../Hooks'
 import Icon from '../../Icons'
+import './Table.css'
```

Comment:
In this project we have used either inline-css or StyledComponent, and avoided defining custom css files, with a few exceptions.
You can see line 41 for an example of StyledComponents use.